example/src/pages/MapBaseLayers.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Select, ViewerContainer } from '@datapunt/asc-ui'
+import { Map } from '@datapunt/react-maps'
+import Controls from '../../../src/components/Zoom'
+import BaseLayer from '../../../src/components/BaseLayer'
+import Scale from '../../../src/components/Scale'
+import {
+  MapLayer,
+  DEFAULT_AMSTERDAM_LAYERS,
+  DEFAULT_AMSTERDAM_MAPS_OPTIONS,
+  AERIAL_AMSTERDAM_LAYERS,
+} from '../../../src/constants'
+
+const StyledMap = styled(Map)`
+  width: 100%;
+  height: 100vh;
+`
+
+const StyledViewerContainer = styled(ViewerContainer)`
+  z-index: 400;
+`
+
+const MapBaseLayers: React.FC = () => {
+  const [baseLayer, setBaseLayer] = useState<string>(
+    DEFAULT_AMSTERDAM_LAYERS[0].urlTemplate,
+  )
+  const handleChange = (
+    e: React.FormEvent<HTMLSelectElement>,
+    layers: MapLayer[],
+  ) => {
+    const { value } = e.currentTarget
+
+    const layer = layers.find(({ id }) => id === value)
+
+    if (layer) {
+      setBaseLayer(layer.urlTemplate)
+    }
+  }
+
+  return (
+    <StyledMap
+      options={{
+        ...DEFAULT_AMSTERDAM_MAPS_OPTIONS,
+      }}
+    >
+      <Scale
+        options={{
+          position: 'bottomright',
+          metric: true,
+          imperial: false,
+        }}
+      />
+      <StyledViewerContainer
+        topLeft={
+          <>
+            <Select
+              label="Varianten"
+              onChange={e => handleChange(e, DEFAULT_AMSTERDAM_LAYERS)}
+            >
+              {DEFAULT_AMSTERDAM_LAYERS.map(({ id, label }) => (
+                <option value={id}>{label}</option>
+              ))}
+            </Select>
+            <Select
```

Comment:
Looks like the label for this select is not automatically linked to the select itself. Perhaps we need to pass in a prop? I would expect this should happen automatically, so perhaps we need to create a bug for this in the components library?

Reply:
ah we can pass the `id`, that will set the `for` on the label and the id on the select